linux_os/guide/system/software/integrity/software-integrity/rpm_verification/rpm_verify_hashes.rule

```diff
@@ -1,7 +1,5 @@
 documentation_complete: true
 
-prodtype: rhel6,rhel7,fedora,ol7
-
```

Comment:
I don't think `rpm_` Rules should have their `prodtypes` removed, at least until there is a mechanism to enable it only for `rpm` based products using data from `product.yml`.

See generated `opensuse` content, its `pkg_manager` is zypper, but the DS contains rule descriptions  with `rpm` commands.
This will become more evident when `ubuntu`, and `debian`  move to `linux_os/`  guide and they start picking up the `rpm` rules.